modules/porous_flow/test/tests/sinks/tests

```diff
@@ -160,6 +160,54 @@
     issues = '#7333'
     design = 'porous_flow/tests/sinks/sinks_tests.md porous_flow/boundaries.md porous_flow/doc/tests/sinks.tex'
   [../]
+  [./s11]
+    type = Exodiff
+    input = s11.i
+    exodiff = s11.e
+    threading = '!pthreads'
+    cli_args = 'BCs/left_T/fluid_phase=0'
+    requirement = 'PorousFlow shall be able to inject fluid at user specified temperature using.'
+    design = 'PorousFlowEnthalpySink.md'
+    issues = '#15742'
+  [../]
+  [./s11_act]
+    type = Exodiff
+    input = s11_act.i
+    exodiff = s11.e
+    threading = '!pthreads'
```

Comment:
do you want a `prereq` here?   Your choice.

Reply:
✅